proxy/ParentSelection.cc

```diff
@@ -570,6 +572,18 @@ ParentRecord::Init(matcher_line *line_info)
         this->ignore_query = false;
       }
       used = true;
+    } else if (strcasecmp(label, "fname") == 0) {
+      // fname=ignore | consider
+      if (strcasecmp(val, "ignore") == 0) {
```

Comment:
I thought the same thing when I read that, but if you look at the surrounding code it's all in the style @pbchou did. I think we should leave this as is. Perhaps a different clean up effort.